ecosystem/sep-0031.md

```diff
@@ -68,6 +68,30 @@ All endpoints respond with content type:
 
 - `application/json`
 
+## Asset Conversions and Liquidity Strategies
+
+This protocol can be used if the Sending and Receiving Client wish to send and receive the same asset, but this is typically not the case. It is more common for the Sending Client to provide the Sending Anchor with a particular asset (the source asset) and have the Receiving Client receive a different asset (the destination asset).
```

Comment:
💡 This paragraph is a little difficult to digest because it leads with the least important information and finishes with the most important information. The most important information is what the protocol is intended to be used for. Suggest reordering:
```suggestion
This protocol is designed for the common case where the Sending Client provides the Sending Anchor with a particular asset (the source asset) and the Receiving Client receives a different asset (the destination asset). It is possible to use this protocol when the Sending and Receiving clients wish to send and receive the same asset, but that is not the common use case it has been designed.
```